datacapture/src/main/java/com/google/android/fhir/datacapture/extensions/MoreQuestionnaireItemComponents.kt

```diff
@@ -61,6 +61,9 @@ internal const val EXTENSION_ITEM_CONTROL_URL_ANDROID_FHIR =
 internal const val EXTENSION_ITEM_CONTROL_SYSTEM_ANDROID_FHIR =
   "https://github.com/google/android-fhir/questionnaire-item-control"
 
+internal const val EXTENSION_DIALOG_URL_ANDROID_FHIR =
+  "https://github.com/google/android-fhir/dialog"
```

Comment:
keep extension urls in the same "namespace". see the location widget extension.

```suggestion
  "https://github.com/google/android-fhir/StructureDefinition/dialog"
```

Reply:
done